libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/Logging/SFLogger.m

```diff
@@ -230,11 +230,11 @@ + (NSString*)loggingIdentifier {
 }
 
 - (void)log:(SFLogLevel)level msg:(NSString *)msg {
-    [self log:level identifier:[self.class loggingIdentifier] format:msg, nil];
+    [self log:level identifier:[self.class loggingIdentifier] format:@"%@", msg];
```

Comment:
My head was circling around this solution, but I hadn't had time to focus on it.  I felt a one-liner was a distinct possibility.  👍